include/mxnet/random_generator.h

```diff
@@ -63,6 +63,10 @@ class RandGenerator<cpu, DType> {
 
     MSHADOW_XINLINE int rand() { return engine_->operator()(); }
 
+    MSHADOW_XINLINE int64_t rand_int64() {
+      return static_cast<int64_t>(engine_->operator()() << 31) + engine_->operator()();
```

Comment:
@leezu over here I use rand_int64() by using mt19937.rand() for 2 32bits

According to the video : https://channel9.msdn.com/Events/GoingNative/2013/rand-Considered-Harmful
is this a right way of doing it truly uniform random integer distribution?

Reply:
Yes, because you're concatenating 2 32bit numbers, where every bit is uniformly random. Thus the resulting 64bits are uniformly random.